### ISSUE TITLE
Searching for terms doesn't return child terms in results

### ISSUE THREAD

**Issue by jasonbahl** [wp-graphql/wp-graphql#598]
Searching for terms doesn't return child terms in the results. 

Here, you can see the term "child category" 

![screen shot 2018-12-04 at 11 23 42 am](https://user-images.githubusercontent.com/1260765/49465864-cf14ec00-f7bb-11e8-9c36-2f13a655a239.png)

And when the following query is executed, I would expect to see that term in the results, but it's not. 

```
{
  categories(where: {search: "child"}) {
    nodes {
      id
      categoryId
      name
    }
  }
```

This is due to the default query arg for root TermObjectConnections being set to `parent => 0`

We should unset the `parent` query arg for search requests to make sure searches account for child terms as well.

**Comment by TylerBarnes** [comment]
If this is implemented https://github.com/wp-graphql/wp-graphql/issues/534 this wont be an issue anymore

**Comment by jasonbahl** [comment]
closed by #597
